business_register/converter/pep.py

```diff
@@ -253,6 +253,7 @@ def __init__(self):
             'pep_id',
             'business_register',
             'CompanyLinkWithPep')
+        self.invalid_data = 0
```

Comment:
maybe? It is counter, not data.
```suggestion
        self.errors_count = 0
```

Reply:
used counter in the naming